### ISSUE TITLE
AsyncCommand use main.stdout / main.stderr

### ISSUE THREAD

**Issue by jakeheis** [kareman/SwiftShell#61]
Right now, `AsyncCommand` will always change the process's stdout and stderr to a `Pipe` (https://github.com/kareman/SwiftShell/blob/master/Sources/SwiftShell/Command.swift#L291). 

There should be a way to start an `AsyncCommand` which still directs output to `main.stdout` and `main.stderr`. Simply forwarding the output using something like:
 ```
command.stdout.onOutput { $0.write(to: &stdout) }
```
is not sufficient, as pipes are block buffered but `main.stdout` is line buffered, so the command's output will not be written to stdout in the expected way.

I think this could potentially be solved by changing `AsyncCommand.init` to something like:

```
init(unlaunched process: Process, stdout: WritableStream? = nil, stderr: WritableStream? = nil)
```

**Comment by kareman** [comment]
Yes that would be useful. But AsyncCommand.init is not public, so we need to change the public api. Do you think the use case is common enough to warrant a new function `runAsyncAndPrint`, which returns a superclass of AsyncCommand without `stdout` and `stderror`?

**Comment by patchthecode** [comment]
Usually one use case is enough to constantly bite you in the ass i always say 😄.
I like this library. I vote yes.

**Comment by kareman** [comment]
@jakeheis I implemented runAsyncAndPrint in branch [61-AsyncCommand_use_main.stdout](https://github.com/kareman/SwiftShell/tree/61-AsyncCommand_use_main.stdout).

**Comment by patchthecode** [comment]
will take pull.

**Comment by kareman** [comment]
Just released version 4.1 with this feature.
